storage/schemadb/src/lib.rs

```diff
@@ -244,20 +252,23 @@ impl DB {
         self.inner.write_opt(db_batch, &default_write_options())?;
 
         // Bump counters only after DB write succeeds.
-        for (cf_name, rows) in rows_locked.iter() {
-            for write_op in rows {
-                match write_op {
-                    WriteOp::Value { key, value } => {
-                        APTOS_SCHEMADB_PUT_BYTES
-                            .with_label_values(&[cf_name])
-                            .observe((key.len() + value.len()) as f64);
-                    },
-                    WriteOp::Deletion { key: _ } => {
-                        APTOS_SCHEMADB_DELETES.with_label_values(&[cf_name]).inc();
-                    },
+        if sampled_kv_bytes {
+            for (cf_name, rows) in rows_locked.iter() {
+                for write_op in rows {
+                    match write_op {
+                        WriteOp::Value { key, value } => {
+                            APTOS_SCHEMADB_PUT_BYTES
+                                .with_label_values(&[cf_name])
+                                .observe((key.len() + value.len()) as f64);
+                        },
+                        WriteOp::Deletion { key: _ } => {
+                            APTOS_SCHEMADB_DELETES.with_label_values(&[cf_name]).inc();
```

Comment:
@msmouse - Not scaling the values here as it creates weird rounding issues for these counters.. its better to scale them in the dashboard itself..

Reply:
Thanks @sitalkedia . K. Change the counter names to intentionally break the dash then. (add _sampled).